.devops/deploy-pipelines.yml

```
@@ -54,18 +47,42 @@ steps:
     echo BLOB_CONTAINER_NAME: $(BLOB_CONTAINER_NAME) 
   displayName: 'Check Site'
 
+# SYNC STORAGE AT EVERY COMMIT, so WITHOUT Enti&Servizi (PIPELINE_TYPE!=Schedule)
 - task: AzureCLI@1
-  displayName: 'Sync Storage'
+  displayName: 'Sync Storage NOT SCHEDULED'
+  condition: ne(variables['Build.Reason'], 'Schedule')
+  inputs:
+    azureSubscription: '$(AZURE_SUBSCRIPTION)'
+    scriptLocation: inlineScript
+    inlineScript: |
+      az storage blob sync --container $(BLOB_CONTAINER_NAME) --account-name $(STORAGE_ACCOUNT_NAME) -s "$(Build.SourcesDirectory)/_site" --exclude-path "assets/json/enti-list-webview.json;assets/json/enti-list-searchable.json;assets/json/enti-list-numbers.json;/app-content/enti-servizi.html;enti/" --exclude-pattern "assets/entijson/*.json"
+
+# SYNC STORAGE AT EVERY SCHEDULE, with Enti&Servizi (PIPELINE_TYPE==Schedule)
+- task: AzureCLI@1
+  displayName: 'Sync Storage ALL'
+  condition: eq(variables['Build.Reason'], 'Schedule')
   inputs:
     azureSubscription: '$(AZURE_SUBSCRIPTION)'
     scriptLocation: inlineScript
     inlineScript: |
       az storage blob sync --container $(BLOB_CONTAINER_NAME) --account-name $(STORAGE_ACCOUNT_NAME) -s "$(Build.SourcesDirectory)/_site"
 
+# PURGE CDN AT EVERY COMMIT, so globally
 - task: AzureCLI@1
-  displayName: 'Purge CDN'
+  displayName: 'Purge CDN NOT SCHEDULED'
+  condition: ne(variables['Build.Reason'], 'Schedule')
```

```suggestion
  condition: eq(variables['Build.Reason'], 'BatchedCI')
```
this is the specific build reason for commits, no blocking

https://docs.microsoft.com/it-it/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables-devops-services